main/freetype/APKBUILD

```diff
@@ -22,11 +22,6 @@ source="http://download.savannah.gnu.org/releases/freetype/freetype-$pkgver.tar.
 	freetype-profile.sh
 	"
 
-# secfixes:
-#   2.7.1-r1:
-#     - CVE-2017-8105
-#     - CVE-2017-8287
-
```

Comment:
alpine-secdb is generated from those comments, so if you remove it from APKBUILD you remove it from alpine-secdb.